### PR TITLE
Update banner for Dart and Flutter related I/O videos

### PR DESCRIPTION
**Pull request by parlough**
Screenshot of staged banner

### CODE REVIEW

File: src/_includes/banner.html [hunk 1]
```diff
@@ -1,9 +1,8 @@
...
   <p class="banner__text">
-    Dart 3 is here with records and patterns, alongside Google I/O!
-    <br>
-    <a href="https://medium.com/p/53f065a10635" class="no-automatic-external">Check out the blog post</a> and 
-    <a href="https://io.google/2023/?utm_source=dart&utm_medium=embedded_marketing&utm_campaign=hpp_banner&utm_content="
-    class="no-automatic-external">register to watch</a> now.
+    Dart 3 is here, with 100% sound null safety, records, and patterns!
+    <a href="https://medium.com/p/53f065a10635" class="no-automatic-external">Learn more</a><br>
+    Learn how Dart and Flutter are both racing forward at Google I/O 2023!
+    <a href="https://io.google/2023/program/?q=keynote,technical-session,workshop,flutter" class="no-automatic-external">Watch now</a>
```
Comment (atsansone) [inline src/_includes/banner.html:6]: **suggestion:** Headlines can be a little choppy. That's OK. Shortening the text.

```suggestion
    Dart 3: Now with 100% sound null safety, records, and patterns!
    <a href="https://medium.com/p/53f065a10635" class="no-automatic-external">Learn more</a><br>
    Racing Forward at I/O 2023 with Flutter and Dart
    <a href="https://io.google/2023/program/?q=keynote,technical-session,workshop,flutter" class="no-automatic-external">Watch the videos!</a>
```

File: src/_includes/banner.html [hunk 2]
```diff
@@ -1,10 +1,11 @@
 <div class="banner">
   <p class="banner__text">
-    Dart 3 is here with 
-    <a href="https://dart.dev/language/records" class="no-automatic-external">records</a>, 
+    Dart 3 is here with
```
Comment (atsansone) [inline src/_includes/banner.html:3]: **issue:** Would prefer to avoid passive voice and the word "here".
```suggestion
    Dart 3 has arrived with
```

**Comment by atsansone** [comment]
Small update.

**Comment by parlough** [comment]
@atsansone Applied, thanks! Back to you for approval :)

**Comment by parlough** [comment]
I think it's fine to not link here anymore since it's value was mostly those few days after.

**Pull request closed without merging**